libs/core/src/lib/package.json

```diff
@@ -19,16 +22,21 @@
     "@angular/animations": "ANGULAR_VER_PLACEHOLDER",
     "@angular/router": "ANGULAR_VER_PLACEHOLDER",
     "@angular/cdk": "ANGULAR_VER_PLACEHOLDER",
-    "rxjs": "RXJS_VER_PLACEHOLDER"
+    "rxjs": "RXJS_VER_PLACEHOLDER",
+    "fundamental-styles": "FDSTYLES_VER_PLACEHOLDER",
+    "@sap-theming/theming-base-content": "THEMING_VER_PLACEHOLDER"
   },
   "dependencies": {
-    "@sap-theming/theming-base-content": "THEMING_VER_PLACEHOLDER",
     "focus-trap": "FOCUSTRAP_VER_PLACEHOLDER",
-    "fundamental-styles": "FDSTYLES_VER_PLACEHOLDER",
     "@fundamental-ngx/i18n": "VERSION_PLACEHOLDER",
     "fast-deep-equal": "FAST_DEEP_EQUAL_VER_PLACEHOLDER",
     "lodash-es": "LODASH_ES_VER_PLACEHOLDER",
     "tslib": "^2.0.0",
     "compare-versions": "COMPARE_VERSIONS_VER_PLACEHOLDER"
+  },
+  "exports": {
+    ".": {
+      "sass": "./styles/fundamental-ngx-core.scss"
```

Comment:
why is this needed?

Reply:
Currently, we have *raw* global styles `libs/core/schematics/ng-add/styles.ts` (fonts, icons, etc.) which we place on the customer side when installing lib with `ng add`. In case tomorrow we need to change these styles, we have to write migration for it. 

If we only be putting link to styles file from our lib, we can change its contents to whatever we want without need to write migrations etc.

Real world example: 
Tomorrow we will add Skeleton feature and these Skeletons will be rendered over the components when applying CSS classes. To have these CSS classes' values loaded, we either have to render a skeleton component somewhere which import these styles (bad approach) or add these styles to global ones (like fonts) so they will be accessible anywhere once global styles are loaded.